core/src/model/LivingEntity.java

```diff
@@ -126,7 +129,7 @@ public void move(Direction direction, float speed) {
     }
 
     /**
-     * Damages every touched hostile entity
+     * Attacks every touched entity
```

Comment:
why is the comment changed? it doesnt do dmg to everyone only hostile ones?

Reply:
Technically it only calls the beAttacked method on every touched entity. The entities determine themselves what the effect of being attacked should be.